ReadME.md

```diff
@@ -143,7 +143,7 @@ This project is part of the [GirlScript Summer of Code 2022](https://gssoc.girls
 | **[Praveen Kumar Purushothaman](https://praveen.science/)**  |
 
 >**Need help?**
-Feel free to contact me at [catsintech@praveen.science](mailto:catsintech@praveen.science).
+Feel free to contact me at [catsintech@praveen.science](mail to:catsintech@praveen.science).
```

Comment:
Hello @chetnaakhandagale 👋 
This should be retained as original to work as expected!